src/libs/actions/Report.js

```diff
@@ -800,21 +812,18 @@ function addAction(reportID, text, file) {
  * network layer handle the delayed write.
  *
  * @param {Number} reportID
- * @param {Number} sequenceNumber
+ * @param {Number} [sequenceNumber]
  */
 function updateLastReadActionID(reportID, sequenceNumber) {
-    const currentMaxSequenceNumber = reportMaxSequenceNumbers[reportID];
-    if (sequenceNumber < currentMaxSequenceNumber) {
-        return;
-    }
```

Comment:
It's not enough to remove this front end check. If the `sequenceNumber` is less than what we already have set in the past we cannot update it. Probably there needs to be an API change to fix this if our intention is to be able to mark an earlier comment as the "last read". See Web-Expensify logic [here](https://github.com/Expensify/Web-Expensify/blob/3dfc29a8f05dc986c8fa86260c69a14f66cda20d/lib/ReportAPI.php#L2851-L2857).

Reply:
I saw it. I plan to update it in a different PR, but it doesn't seem to affect the "visible" functionality 🤷

Reply:
Yep! I planned to create the PR later, since it doesn't affect the bulk of the flow, but might as well get it out of the way now: https://github.com/Expensify/Web-Expensify/pull/30753